packages/api/src/controllers/stripe.ts

```diff
@@ -209,19 +209,32 @@ const sendUsageRecordToStripe = async (
             quantity: parseInt(overUsage.DeliveryUsageMins.toFixed(0)),
             timestamp: Math.floor(new Date().getTime() / 1000),
             action: "set",
-          }
+          },
         );
       } else if (product.name === "Storage") {
-        await stripe.subscriptionItems.createUsageRecord(
-          subscriptionItemsByLookupKey["tstorage_usage"],
-          {
-            quantity: parseInt(overUsage.StorageUsageMins.toFixed(0)),
-            timestamp: Math.floor(new Date().getTime() / 1000),
-            action: "set",
-          }
-        );
+        try {
+          await stripe.subscriptionItems.createUsageRecord(
+            subscriptionItemsByLookupKey["tstorage_usage2"],
+            {
+              quantity: parseInt(overUsage.StorageUsageMins.toFixed(0)),
+              timestamp: Math.floor(new Date().getTime() / 1000),
+              action: "set",
+            },
+          );
+        } catch (e) {
+          // Fallback to old storage lookup key if new one fails.
+          // To avoid breaking the usage reporting during migration
```

Comment:
Will there be an exception if the other lookup key doesn't exist? afaik the lookup will just return `undefined`

Reply:
Yeah you're right, no exception, just a bad status code from stripe. I think we can just migrate without this try catch then